cmake/FindQDeclarative.cmake

```diff
@@ -0,0 +1,40 @@
+# Try to find qdeclarative5-boostable
+# Once done this will define
+# QDECLARATIVE_FOUND - System has qdeclarative
```

Comment:
QDECLARATIVE is also a bit misleading, it evokes QtDeclarative which is basically QML but what you are actually searching for is mdeclarativecache5 which is part of mapplauncherd-qt5.

MAPPLAUNCHERD_QT5_FOUND would be a better name